### ISSUE TITLE
include_experimental does not work when set via metadata

### ISSUE THREAD

**Issue by dcopso** [aws-cloudformation/cfn-lint#2773]
### CloudFormation Lint Version

0.77.10

### What operating system are you using?

Ubuntu 22

### Describe the bug

The README.md file says that `include_experimental` can be enabled with the `-e` flag or via metadata: https://github.com/aws-cloudformation/cfn-lint/blob/v0.77.10/README.md#parameters

However, using metadata to enable `include_experimental` does not have the same behavior as `-e`:
```shell
$ yq -y .Metadata repro.yml 
cfn-lint:
  config:
    include_experimental: true

$ cfn-lint repro.yml
# no failures

$ cfn-lint -e repro.yml 
W3037 Invalid permission "ListObjects" for "s3" found in permissions
repro.yml:12:13
```

Thanks.

### Expected behavior

Expected output:
```
W3037 Invalid permission "ListObjects" for "s3" found in permissions
```

### Reproduction template

```yaml
Metadata:
  cfn-lint:
    config:
      include_experimental: true
Resources:
  BadPolicy:
    Type: AWS::IAM::ManagedPolicy
    Properties:
      PolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Effect: Allow
            Action:
              - s3:ListObjects
            Resource:
              - 'arn:aws:s3:::bucket_name'
```
